dftimewolf/lib/containers/containers.py

```diff
@@ -111,3 +111,54 @@ def __init__(self, type_, name, value):
     self.type = type_
     self.name = name
     self.value = value
+
+
+class File(interface.AttributeContainer):
+  """Attribute container definition for generic files.
+
+  Attributes:
+    name (str): Name of the attribute.
+    path (str): Value of the attribute.
+  """
+  CONTAINER_TYPE = 'file'
+
+  def __init__(self, name, path):
+    """Initializes the attribute.
+
+    Args:
+      name (str): Name of the attribute.
+      path (str): Value of the attribute.
+    """
+    super(File, self).__init__()
+    self.name = name
+    self.path = path
+
+class ForensicsVM(interface.AttributeContainer):
+  """Attribute container definition for a forensics virtual machine.
+
+  Attributes:
+    name (str): Identifying name for the virtual machine
+    evidence_disk (Object): The disk containing the forensic evidence.
+    platform (str): The cloud platform where the VM is located. One of
+        {gcp,aws,azure}.
+  """
+  CONTAINER_TYPE = 'forensics_vm'
+
+  def __init__(self, name, evidence_disk, platform):
+    super(ForensicsVM, self).__init__()
+    self.name = name
+    self.evidence_disk = evidence_disk
+    self.platform = platform
+
+
+class URL(interface.AttributeContainer):
```

Comment:
URL or URI? I see in the gce_disk_export there's code like this: "containers.URL(path=output_uri)". I suspect these should all be URL